### PR TITLE
[BEAM-7601] fixed typo

### PR DESCRIPTION
**Pull request by brucearctor**
In https://beam.apache.org/community/in-person/, under Meetups:

Actual: occassionally

Expected: occasionally

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_VR_Flink/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_VR_Spark/lastCompletedBuild/)
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Batch/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Streaming/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Spark_Batch/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python3_Verify/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/) <br> [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python_PVR_Flink_Cron/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_VR_Spark/lastCompletedBuild/)

Pre-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

--- |Java | Python | Go | Website
--- | --- | --- | --- | ---
Non-portable | [Build Status](https://builds.apache.org/job/beam_PreCommit_Java_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Go_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Website_Cron/lastCompletedBuild/) 
Portable | --- | [Build Status](https://builds.apache.org/job/beam_PreCommit_Portable_Python_Cron/lastCompletedBuild/) | --- | ---

See [.test-infra/jenkins/README](https://github.com/apache/beam/blob/master/.test-infra/jenkins/README.md) for trigger phrase, status and link of all Jenkins jobs.

### CODE REVIEW

**Comment by brucearctor** [comment]
@robertwb

**Comment by brucearctor** [comment]
R: @mxm

File: website/src/community/in-person.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ While the official communication happens on the mailing list, and you can find u
 
 ## Meetups
 
-We occassionally meetup in various locations around the globe.  Active or to-be-started meetups include:
+We occasionally meetup in various locations around the globe.  Active or to-be-started meetups include:
```
Comment (mxm) [inline website/src/community/in-person.md]:
```suggestion
We occasionally meetup in various locations around the globe. Active or to-be-started meetups include:
```

Comment (brucearctor) [reply website/src/community/in-person.md]: What is beam convention?  Is everything supposed to be single space?  Do we need new issues for all of the rest of site?

Comment (brucearctor) [reply website/src/community/in-person.md]: OK.

Comment (brucearctor) [reply website/src/community/in-person.md]: either way fine

Comment (mxm) [reply website/src/community/in-person.md]: I think we normally use a single space after a period but this file seems to use double spaces. Fine with me.

**Comment by brucearctor** [comment]
@mxm - fixed rest of page to make it consistent (single spaces abound)

**Comment by mxm** [comment]
Thank you @brucearctor. Will squash and merge after the tests pass.

**Comment by brucearctor** [comment]
Run Website PreCommit

**Comment by mxm** [comment]
Retest this please

**Comment by mxm** [comment]
Github web hook issue have been resolved.

**Comment by mxm** [comment]
Could you squash the three commits into a single commit, then force push? If I use the "Squash and merge" feature of Github, it will change the author to my name, which would be a shame :)

**Comment by mxm** [comment]
Just did the squashing manually on your branch. Thanks @brucearctor!

**Pull request merged**